.github/workflows/clang-nightly.yaml

```diff
@@ -20,8 +20,8 @@ concurrency:
 jobs:
   clang-dev:
     name: Build with clang nightly
-    runs-on: ubuntu-latest
     if: github.repository == 'scylladb/scylladb'
+    runs-on: ubuntu-latest
```

Comment:
This looks like a mis-squash: this hunk effectively corrects a small mistake in patch#1; therefore, it should be squashed into patch#1, not patch#2.

Reply:
resolved in v2, thanks